.cirrus.yml

```diff
@@ -743,3 +743,43 @@ task:
     - make build config=debug
   stress_test_script:
     - make ${TARGET} config=debug
+
+task:
+  only_if: $CIRRUS_CRON == "stress"
+
+  timeout_in: 120m
+
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-monterey-xcode:13.3.1
+
+  matrix:
+    - name: "Stress Test: arm64 macos monterey [release]"
+      environment:
+        TARGET: test-stress-release
```

Comment:
Do these not need the `IMAGE` attribute like the Linux tests?

Reply:
No because image is part of the cache busting for linux where there are various images but there's only one for this so the two places it exists are in the cache busting string and the selection, rather than needing the indirection of IMAGE.